### PR TITLE
Added checks for user data going into eval and exec.

### PR DESCRIPTION
**Pull request by minusworld**
Addresses https://github.com/returntocorp/sgrep-rules/issues/260

### CODE REVIEW

File: python/django/security/user-eval-format-string.yaml [hunk 1]
```diff
@@ -0,0 +1,133 @@
+rules:
+  - id: user-eval-format-string
+    patterns:
+      - pattern-inside: |
+          def $F(...):
+            ...
+      - pattern-either:
```
Comment (ulziibay) [inline python/django/security/user-eval-format-string.yaml]: you can add the f-string variants of these checks in one go

File: python/django/security/user-exec.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+          - pattern: |
+              $V = request.$W[...]
+              ...
+              $A = exec(..., $V, ...)
+          - pattern: |
+              $V = request.$W[...]
+              ...
+              return exec(..., $V, ...)
+    message: |
+      Found user data in a call to 'exec'. This is extremely dangerous.
```
Comment (ulziibay) [inline python/django/security/user-exec.yaml]: Add maybe a link about RCE

File: python/django/security/user-eval-format-string.yaml [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+              ...
+              $A = eval(..., $S, ...)
+          - pattern: |
+              $V = request.$W[...]
+              ...
+              $S = $STR.format(..., $V, ...)
+              ...
+              return eval(..., $S, ...)
+    message: |
+      Found user data in a call to 'eval'. This is extremely dangerous.
```
Comment (ulziibay) [inline python/django/security/user-eval-format-string.yaml]: Maybe add a link about RCE

File: python/django/security/user-exec.yaml (hunk 2 above)
Comment (minusworld) [reply python/django/security/user-exec.yaml]: Roger. That's a good idea.

**Pull request merged**